### DISCUSSION TITLE
Unable to Install Big Sur

### DISCUSSION THREAD

**Discussion by bocah-ndeso** [SkyrilHD/Dell-E7250-Hackintosh#25] [General]
I used the EFI copy provided in this repository, it worked fine for booting. Detected keyboard, screen, touch screen and trackpad. But never success on installation of Big Sur. Is there special steps needed in order to install Big Sur?

Thanks

**Comment by nunnethan** [comment]
What do you mean? Do you have any specific error messages? Have you formatted your drive as APFS and turned SATA mode to ACPI in your BIOS settings?
